Mathlib/Analysis/Meromorphic/Order.lean

```diff
@@ -132,6 +132,26 @@ theorem order_mul {f g : 𝕜 → 𝕜} {x : 𝕜} (hf : MeromorphicAt f x) (hg
     (hf.mul hg).order = hf.order + hg.order :=
   hf.order_smul hg
 
+/-- The order of the inverse is the negative of the order. -/
+theorem order_inv {f : 𝕜 → 𝕜} {z₀ : 𝕜} (hf : MeromorphicAt f z₀) :
+    hf.order = -hf.inv.order := by
+  -- Trivial case: f locally zero
```

Comment:
Personally, I think I would drop the comment. But I'm also happy with keeping it. In the latter case, please add a comment for the non-trivial case as well.

Reply:
Comment dropped.